lightspark/lightspark_client.py

```diff
@@ -933,6 +995,13 @@ def _hash_phone_number(self, phone_number_e164_format: str) -> str:
             )
         return sha256(phone_number_e164_format.encode()).hexdigest()
 
+    def hash_uma_identifier(self, identifier: str, signing_private_key: bytes) -> str:
+        now = datetime.now(timezone.utc)
+        input_data = (
+            identifier + f"{now.month}-{now.year}" + signing_private_key.decode("utf-8")
```

Comment:
hmm sorry if I didn't notice this in other PRs, but should we be hex-encoding the private key instead of utf-8? I'm not entirely sure if this is always safe or not.

Reply:
ah yeah thats true 😢 will update